### ISSUE TITLE
Validation: Move error message box to the bottom 

### ISSUE THREAD

**Issue by mrosvik** [Altinn/app-frontend-react#100]
## Description
_As a user I want to see why I can´t click "Next", so that I can complete the form._ 

- The error message box thats shows when you try to hit "Next"/"Send" should be connected with the "Next"/"Send"-button, since the summary explain why the next button is not working. 
- The error messages should link to the field where the error is (set focus).
- WCAG: The current error message use H2 and H4 as texts. This is not compliant with WCAG. Remember to think about what header we use on the new component

## Design
https://www.figma.com/file/Pvti3aRcOwH0k5Z7DrLznk/Arbeidsomr%C3%A5de%3A-Altinn-Studio-APPS?node-id=2545%3A50628

<img width="1847" alt="image" src="https://user-images.githubusercontent.com/1464915/158781837-f24304be-4f57-4bf4-9251-ca1f853a612e.png">

## Development tasks
- [ ] Move error box from the top, into the bottom of the form. 
- [ ] Link the error to the connected field (set focus).

## Definition of done

- [ ] Documentation (docs.altinn.studio) is updated (if relevant)
  - [ ] Technical documentation
  - [ ] User documentation
- [ ] QA
- [ ] Manual test is complete (if relevant)
- [ ] Automated test is implemented (if relevant)
- [ ] All tasks in this userstory are closed (i.e. remaining tasks are moved to other user stories or marked obsolete)

**Comment by olemartinorg** [comment]
Innspill fra @FinnurO: Mulig vi bør endre teksten fra "Du må *fikse* disse feilene før du kan gå videre" til "Du må *rette* disse feilene før du kan gå videre".

**Comment by Febakke** [comment]
@olemartinorg Is this the correct place for test feedback

**Screen reader test:** Ok (VoiceOver on chrome)
**Keyboard navigation:** Ok

**Smaller screens:** 
On smaller screens the text fall down and the icon remains. It would be better to have the text linebreak and always have the start of the text aligned with the icon.
![Skjermbilde 2022-08-02 kl  10 59 26](https://user-images.githubusercontent.com/33222679/182355568-8b325c4f-8ae6-4f0b-82d1-913d238feb47.png)

**Contrast**
In the Figma sketches it looks the pink color is set to 50% opacity. With a white background this would equal to #FCE4E9. This is probably easy to fix with tokens at a later stage?

In the design process we have overlooked a contrast issue when the green (Submit) button are visible on the last page of a application. The button contrast with the red background is to low and break with [1.4.11 Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) I could not find an easy way out if this with the current design. It might be best if I create a new issue to work on how to solve this from our end?

**hover**
Not specified in the sketches and not a WCAG breach. But I think it is a little off on desktop to not have any hover state on the links in the error summary. Would be easily solved with an underline on hover. We have hover states on all other inputs and buttons so it sticks out.

**Comment by mrosvik** [comment]
In addition to what @Febakke mention, the hover border should be black, 2px and cover borders of any extra links appearing in the message (so there is no double borders).
![image](https://user-images.githubusercontent.com/1464915/183883848-c485a83b-d428-4af8-80a6-feb1861b2c61.png) 

(Talked with @olemartinorg about this) :) 

The contrast on buttons will be fixed when the latest design tokens are added to the button component.

**Comment by olemartinorg** [comment]
I looked into doing this in CSS, but it turns out to be a lot more hacky than I'd like. I looked into it, and with the changes in #372 I moved the code to render links in error messages, so after that is merged I can quite easily extend the function to remove rendering of links inside these error messages (only when shown in this bottom error report). That should solve the link issues without any hacks:

![20220810_16h52m07s_grim](https://user-images.githubusercontent.com/700139/183935217-6077da78-5e7a-4236-98e9-6096b170c213.png)
